src/app/qgisapp.cpp

```diff
@@ -3544,6 +3545,20 @@ void QgisApp::refreshProfileMenu()
   QAction *newProfileAction = mConfigMenu->addAction( tr( "New Profile…" ) );
   newProfileAction->setObjectName( "mActionNewProfile" );
   connect( newProfileAction, &QAction::triggered, this, &QgisApp::newProfile );
+
+  mConfigMenu->addSeparator( );
+
+  QAction *openProfileSelectionDialog = mConfigMenu->addAction( tr( "Open Profile Selector…" ) );
+  openProfileSelectionDialog->setObjectName( "mActionOpenProfileSelector" );
+  connect( openProfileSelectionDialog, &QAction::triggered, this, [this, activeName]()
+  {
+    auto dlg = QgsUserProfileSelectionDialog( userProfileManager(), activeName );
+
+    if ( dlg.exec() == QDialog::Accepted )
+    {
+      userProfileManager()->loadUserProfile( dlg.selectedProfileName() );
+    }
```

Comment:
QGIS coding convention is to avoid `auto` for anything but iterator/heavily template based types:

```suggestion
    QgsUserProfileSelectionDialog dlg( userProfileManager(), activeName );

    if ( dlg.exec() == QDialog::Accepted )
    {
      userProfileManager()->loadUserProfile( dlg.selectedProfileName() );
    }
```

Reply:
My bad, fixed.